src/main/java/ecdar/abstractions/QueryType.java

```diff
@@ -29,26 +28,23 @@ public String getSymbol() {
 
     public static QueryType fromString(String s) {
         switch (s.toLowerCase()) {
+            case "reachability":
+                return REACHABILITY;
             case "refinement":
                 return REFINEMENT;
-            case "quotient":
-                return QUOTIENT;
             case "specification":
                 return SPECIFICATION;
             case "implementation":
                 return IMPLEMENTATION;
             case "consistency":
+                return CONSISTENCY;
             case "local-consistency":
                 return LOCAL_CONSISTENCY;
-            case "global-consistency":
-                return GLOBAL_CONSISTENCY;
             case "bisim":
```

Comment:
Shouldn't this be "bisim-minim" too?
```suggestion
            case "bisim-minim":
```

Reply:
Whoops :sweat_smile: